ci/upload_gcs_artifact.sh

```diff
@@ -18,9 +18,28 @@ if [ ! -d "${SOURCE_DIRECTORY}" ]; then
   exit 1
 fi
 
-BRANCH=${SYSTEM_PULLREQUEST_PULLREQUESTNUMBER:-${BUILD_SOURCEBRANCHNAME}}
-GCS_LOCATION="${GCS_ARTIFACT_BUCKET}/${BRANCH}/${TARGET_SUFFIX}"
+if [[ "$TARGET_SUFFIX" == "docs" ]]; then
+    # docs upload to the last commit sha (first 7 chars) in the developers branch
+    UPLOAD_PATH="$(git log --pretty=%P -n 1 | cut -d' ' -f2 | head -c7)"
```

Comment:
This is correct in PullRequest context but not in PostSubmit context:

e.g.
This commit generated https://storage.googleapis.com/envoy-postsubmit/7be58c8/docs/index.html while it shouldn't be that.

Reply:
@lizan i guess my question is do the docs artifacts need to be published in post-submit ?

if so i can update, i assume you want it with just the PR  path ?